README.md

```diff
@@ -85,44 +201,110 @@ You can use OpenCensus Web via a `<script>` tag that makes uses of the
 The script tag should go before the `</body>`. You will also need to set the
 `ocAgent` endpoint in a different `<script>` and may also specify the 
 trace sample rate via an `ocSampleRate` global variable as well.
-```html
-...
-  <script src="https://unpkg.com/@opencensus/web-scripts@0.0.3/dist/initial-load-all.js"
-          integrity="sha384-VPY9XX7tiXeLekDPFXkfO2AqNpLTCNOzfXxVghzoVP05PXrG+wtHOW2kOP2ggO9o"
-          async crossorigin="anonymous">
-  </script>
-  <script>
-    // HTTP endpoint of the OpenCensus Agent you ran and exposed to the internet
-    // in Step 1 above.
-    ocAgent = 'https://example.com/my-opencensus-agent-endpoint';
 
-    // Sample all requests for trace, which is useful when testing.
-    // By default this is set to sample 1/10000 requests.
-    ocSampleRate = 1.0; 
-  </script>
-</body>
-```
-
-You can also host the `initial-load-all.js` bundle from your own site. For
+- If you only want to use the **initial page load** functionality:
+  ```html
+  ...
+    <script>
+      // HTTP endpoint of the OpenCensus Agent you ran and exposed to the internet
+      // in Step 1 above.
+      ocAgent = 'https://example.com/my-opencensus-agent-endpoint';
+
+      // Sample all requests for trace, which is useful when testing.
+      // By default this is set to sample 1/10000 requests.
+      ocSampleRate = 1.0;  
+    </script>
+    <script src="https://unpkg.com/@opencensus/web-scripts@0.0.3/dist/initial-load-all.js"
+            integrity="sha384-VPY9XX7tiXeLekDPFXkfO2AqNpLTCNOzfXxVghzoVP05PXrG+wtHOW2kOP2ggO9o"
+            async crossorigin="anonymous">
+    </script>
+  </body>
+  ```
+
+- In the other side, in case you want to **trace user interactions**, you have two options:
+  In addition to the previously explained variables, you need to set the `ocTraceHeaderHostRegex`
+  regex to tell the OC Web which HTTP requests can be intercepted in a interaction to send the 
+  *Trace Context Header*.
+
+  ```html
+    ...
+    <script>
+      // HTTP endpoint of the OpenCensus Agent you ran and exposed to the internet
+      // in Step 1 above.
+      ocAgent = 'https://example.com/my-opencensus-agent-endpoint';
+
+      // Sample all requests for trace, which is useful when testing.
+      // By default this is set to sample 1/10000 requests.
+      ocSampleRate = 1.0;
+      
+      // Send the trace header to all hosts.
+      window.ocTraceHeaderHostRegex = /.*/;
+    </script>
+    ...
+  ```
+
+  - If your application already uses the `Zone.js` library, this being the case for non `Angular` apps: 
```

Comment:
Should this say "if your application does not already ..."? Should we say "the case for non-Angular apps (e.g. React, etc.)"?